cogeo_mosaic/scripts/cli.py

```diff
@@ -77,7 +77,7 @@ def create(
     quiet,
 ):
     """Create mosaic definition file."""
-    input_files = input_files.read().splitlines()
+    input_files = filter(None, input_files.read().splitlines())
```

Comment:
I think `input_files.read().strip()` would also work

Reply:
```bash
$ cat test.txt 
1.tif

2.tif
3.tif

4.tif


```

```python
$ python
>>> f = open("test.txt")
>>> f.read().strip().splitlines()
['1.tif', '', '2.tif', '3.tif', '', '4.tif']
>>> f = open("test.txt")
>>> list(filter(None, f.read().splitlines()))
['1.tif', '2.tif', '3.tif', '4.tif']
```

Reply:
🤦 this will remove only empty line at the end of the file